.rubocop.yml

```diff
@@ -12,6 +12,10 @@ ClassAndModuleChildren:
 SignalException:
   Enabled: false
 
+Metrics/MethodLength:
+  Exclude:
+    - 'apps/web/views/**/*'
```

Comment:
Tsktsktsk

Reply:
This is necessary because of the way that forms work in Hanami!